docs/source/guide/distributed-apis.rst

```diff
@@ -258,18 +258,22 @@ the same as single-process sampling.
 Split workloads
 ~~~~~~~~~~~~~~~
 
-Users need to split the training set so that each trainer works on its own subset. Similarly,
+To train a model, users first need to split the dataset into training, validation and test sets.
+For distributed training, this step is usually done before we invoke :func:`dgl.distributed.partition_graph`
+to partition a graph. We recommend to store the data split in boolean arrays as node data or edge data.
+For node classification tasks, the length of these boolean arrays is the number of nodes in a graph
```

Comment:
Is this a heterogeneous graph or this should be a homogeneous graph? Do we have any context to tell the users the partition can only work on homogeneous graph and if they want to partition a heterogeneous graph they need to do blablabla

Reply:
this is about data split for training/validation/testing. it's not about graph partition. i think graph partition is explained in another section.